dist/yoast-seo.js

```diff
@@ -2182,13 +2186,17 @@ YoastSEO.SnippetPreview.prototype.getPeriodMatches = function() {
 
 /**
  * formats the keyword for use in the snippetPreview by adding <strong>-tags
+ * strips unwanted characters that could break the regex or give unwanted results
  * @param textString
  * @returns textString
  */
 YoastSEO.SnippetPreview.prototype.formatKeyword = function( textString ) {
 
+	// removes characters from the keyword that could break the regex, or give unwanted results.
+	var keyword = this.refObj.rawData.keyword.replace( /[\-\[\]\/\{\}\(\)\*\+\?\.\\\^\$\|]/g, "" );
```

Comment:
Is it possible to save this in a class variable so you don't need to set the regex everytime? In this file alone, this exact regex occurs 3 times. If you need to change the regex, you have to change it in 3 spots as well. It'd be better to have 1 place where this is set and changed.